expirationd.lua

```diff
@@ -289,11 +276,22 @@ local function get_task(name)
 end
 
 -- default process_expired_tuple function
-local function default_tuple_drop(space_id, args, tuple)
-    box.space[space_id]:delete(construct_key(space_id, tuple))
+-- task as parameter was added to preserve backward compatibility
```

Comment:
From this https://github.com/tarantool/expirationd/pull/52#discussion_r631687664 it follows that we should leave the first two input parameters, although they are not used, since expiration_process passes them to this function.

Reply:
Still don't get why we need to keep the old parameter list in the private function.

Reply:
Question is outdated after verbal discussion